.env.example

```diff
@@ -1,4 +1,4 @@
 # The ID of your GitHub integration
-INTEGRATION_ID=
-
+APP_ID=
 WEBHOOK_SECRET=development
+WEBHOOK_PROXY_URL=https://smee.io/xxxxxxxxx
```

Comment:
Might want to comment this out by defaul and include some docs for creating a new channel. See probot/template for an example